gBrogue Source/PlatformCode/main.c

```diff
@@ -22,6 +24,11 @@ struct brogueConsole currentConsole;
 
 boolean serverMode = false;
 boolean noMenu = false;
+boolean noRestart = false;
+boolean noScores = false;
+boolean noRecording = false;
+boolean noSaves = false;
```

Comment:
These are a series of new command line arguments that turn off functionality inappropriate for web-brogue. They can be used for the non-web version just as extra configurability.